README.md

```diff
@@ -2,9 +2,10 @@
   <img src="docs/tersets.jpg" alt="TerseTS">
 </h1>
 
-TerseTS is a library that provides methods for lossless and lossy compressing time series. To match existing literature the methods are organized based on [Time Series Compression Survey](https://dl.acm.org/doi/10.1145/3560814). The library is implemented in Zig and provides a C-API with [bindings](#Installation) for other languages.
+TerseTS is a library that provides methods for lossless and lossy compressing time series. To match existing literature the methods are organized based on [Time Series Compression Survey](https://dl.acm.org/doi/10.1145/3560814). The library is implemented in Zig and provides a C-API with [bindings](#usage) for other languages.
```

Comment:
Maybe change to "... and provides a Zig-API and a C-API...." to make it clear that both exists.

Reply:
Fixed.